yaksh/views.py

```diff
@@ -954,3 +976,45 @@ def design_questionpaper(request):
         context = {'form': form, 'questionpaper':True}
         return my_render_to_response('yaksh/design_questionpaper.html',
                                      context, context_instance=ci)
+
+@login_required
+def view_profile(request):
+    """ view moderators and users profile """
+
+    user = request.user
+    return my_render_to_response('yaksh/view_profile.html', {'user':user})
+
+@login_required
+def edit_profile(request):
+    """ edit profile details facility for moderator and students """
+
+    context = {}
+    user = request.user
+    ci = RequestContext(request)
+
+    if not hasattr(user, 'profile'):
+        new_form = EditProfile(request.POST, user=user)
+        form = EditProfile(user=user)
+    else:
+        profile = Profile.objects.get(user_id=user.id)
+        new_form = EditProfile(request.POST, user=user, instance=profile)
+        form = EditProfile(user=user, instance=profile)
+
+    if request.method == 'POST':
```

Comment:
if POST then check for profile instance and then new_form = EditProfile(request.POST,..)
because new_form and form  in the **if not hasattr **is slightly confusing